editors/neovim/Portfile

```
@@ -48,7 +48,13 @@ cmake.build_type        Release
 configure.args-append   -DLUA_PRG=${prefix}/bin/luajit
 
 # Building parsers is normally an extra step, see https://github.com/neovim/neovim/issues/29042
-patchfiles              embed-parsers-build.diff
+patchfiles              embed-parsers-build.diff  \
+                        patch-lua51-includes.diff
+
+post-patch {
+    reinplace "s|@PREFIX@|${prefix}|g" \
+    ${worksrcpath}/CMakeLists.txt
```

Line 56 should be indented more than line 55. You could also merge the two lines like so:

```suggestion
    reinplace "s|@PREFIX@|${prefix}|g" ${worksrcpath}/CMakeLists.txt
```